build/flatpak/metainfo.xml.in

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is not flatpak-specific. It's called AppStream and many distributions also use it to describe packages in their native repos.

See https://wiki.debian.org/AppStream

You might want to keep it out of the `flatpak/` subdirectory and also ship it is source release tarballs.